build-linux.sh

```diff
@@ -15,12 +15,23 @@ sudo mkdir /opt/cmake
 sudo sh cmake-$version.$build-Linux-x86_64.sh --prefix=/opt/cmake --skip-license
 sudo ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
 
+# add flat buffers
+cd ~
+git clone https://github.com/google/flatbuffers.git
+cd ./flatbuffers
+git checkout 925c1d77fcc72636924c3c13428a34180c30f96f  # 1.10.0 release commit
+mkdir build
+cd build
+cmake .. -DFLATBUFFERS_BUILD_TESTS=Off -DFLATBUFFERS_INSTALL=On -DCMAKE_BUILD_TYPE=Release -DFLATBUFFERS_BUILD_FLATHASH=Off
+make -j4
+sudo make install
+
 cd /reinforcement_learning
 
 # Build reinforcement_learning
 mkdir -p build
 cd build
-cmake .. -DTURN_OFF_DEVIRTUALIZE=On
+cmake .. -DTURN_OFF_DEVIRTUALIZE=On -DCMAKE_PREFIX_PATH=/root/flatbuffers/CMake
```

Comment:
Why do we need this if it has been installed?

Reply:
Not needed.  Removed.